theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -296,7 +296,7 @@ def test_all(self):
                     for b in self.border_modes:
                         try:
                             self.tcase(i, f, s, db, dflip,
-                                       dprovide_shape, fd)
+                                       dprovide_shape)
```

Comment:
Yeah I thought so. I've explained why i did that here, https://github.com/Theano/Theano/pull/4587#issuecomment-230591403